cfgov/es_mappings/calendar_event.json

```diff
@@ -0,0 +1,59 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "all_day": {
+      "type": "boolean"
+    },
+    "calendar": {
+      "type": "string",
+      "index": "not_analyzed"
+    },
+    "created": {
+      "type": "date",
+      "format": "dateOptionalTime"
+    },
+    "date": {
+      "type": "date",
+      "format": "dateOptionalTime"
+    },
+    "day": {
+      "type": "date",
+      "format": "dateOptionalTime"
+    },
+    "description": {
+      "type": "string"
+    },
+    "dtend": {
```

Comment:
Since `date` is spelled out earlier, would it make sense to have `date_end`, `data_start`, and `date_stamp` here and on L:32 and L:36?